### ISSUE TITLE
Null reference for link property

### ISSUE THREAD

**Issue by BernieWhite** [microsoft/PSRule#1393]
**Description of the issue**

Null reference exception is raised with a empty `link` property when `link` is not set for YAML / JSON rules.

**Module in use and version:**

- Module: PSRule
- Version: **2.7.0-B0097**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      7.3.1
PSEdition                      Core
GitCommitId                    7.3.1
OS                             Microsoft Windows 10.0.22621
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
